utilsplot.py

```diff
@@ -601,19 +613,22 @@ def rga_plot(G, w_start=-2, w_end=2, axlim=None, points=1000, fig=0,
                 box = ax.get_position()
                 ax.set_position([box.x0, box.y0,
                                  box.width * 0.8, box.height * 0.9])
-
-    elif plot_type == 'outputs':  # i
+    
+    elif plot_type in plotting_order.keys():
```

Comment:
`in dictionary` does the same as `in dictionary.keys()`

Reply:
**I was unaware, corrected.**